std/algorithm/comparison.d

```diff
@@ -61,7 +61,7 @@ import std.range.primitives;
 import std.traits;
 // FIXME
 import std.typecons; // : tuple, Tuple, Flag;
-import std.meta : allSatisfy;
+import std.meta; // : allSatisfy, anySatisfy;
```

Comment:
So when the transition period for issues 313/314 is over, all of the imports can be fixed by just removing the comments. This has been done to a lot of files recently.